src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/ProjectSystem/ProjectState.cs

```diff
@@ -32,16 +37,21 @@ internal sealed class ProjectState
         ProjectDifference.DocumentAdded |
         ProjectDifference.DocumentRemoved;
 
-    private static readonly ImmutableDictionary<string, DocumentState> s_emptyDocuments = ImmutableDictionary.Create<string, DocumentState>(FilePathNormalizingComparer.Instance);
-    private static readonly ImmutableDictionary<string, ImmutableArray<string>> s_emptyImportsToRelatedDocuments = ImmutableDictionary.Create<string, ImmutableArray<string>>(FilePathNormalizingComparer.Instance);
+    private static readonly ImmutableDictionary<string, DocumentState> s_emptyDocuments
```

Comment:
❓why not just use `ImmutableDictionary<string, DocumentState>.Empty` here?

Reply:
oh, answering my own question but leaving here in case others might wonder: It's so the comparer is correct on mutations

Reply:
That's right! We don't want to create a new empty `ImmutableDictionary<,>` with the correct comparer for each new project.